src/components/SystemMenu/SystemMenu.jsx

```diff
@@ -65,6 +65,18 @@ function SystemMenu({ showTutorial }) {
     api.executeLuaScript(script);
   }
 
+  async function openGuiInBrowser() {
+    if (!luaApi) { return; }
+    const portProperty = await luaApi.propertyValue('Modules.WebGui.Port');
```

Comment:
If this is a property, shouldn't we already have it in the property tree? Might be more performant to use useSelector than to wait for the engine to pass the value

Reply:
Sure.  Just did it the same way here as we do a bit further up in the code, for consistency. But I'm ok with changing it to a using useSelector for the other places in this file as well